cfme/infrastructure/virtual_machines.py

```diff
@@ -8,17 +8,27 @@
 import attr
 import fauxfactory
 import re
-from navmazing import NavigateToSibling, NavigationDestinationNotFound, NavigateToAttribute
+from navmazing import (
```

Comment:
This look like you're applying black formatting to the file while you're making changes. Our pre-commit is currently setup to exclude the cfme/ folder from black formatting. We'll apply it all at once in a PR that doesn't try to change functionality, just formatting.

Trying to apply this formatting here makes it much more difficult to identify and review your actual functional changes to the module, and I think it would be better to use the pre-commit config that comes with integration_tests, and not apply it individually like this.

Reply:
Thanks for the explanation Mike. I was just playing around trying to figure things out.
Will change this PR as suggested.

Reply:
No worries - I like how you're being pro-active!  We'll be able to get this reviewed and merged quicker with focused changes though :)